tests/scripts/all.sh

```diff
@@ -2479,6 +2479,105 @@ component_test_psa_crypto_config_accel_ecc () {
     make test
 }
 
+component_test_verify_ec_max_size () {
```

Comment:
Naming nit: I'd prefer something in the `component_test_psa_crypto_config_accel_` space. Perhaps something like `component_test_psa_crypto_config_accel_ecc_curves_except_p192` and then the other one coming later could be `accel_ecc_curves_except_x25519`? (P-192 is NIST's name for secp192r1, picked it here because it's shorter.) Or `accel_all_curves_except_xxx`?